Dockerfile

```diff
@@ -1,18 +1,29 @@
-FROM ghcr.io/wepublish/node:16.1 as production
+FROM node:16-alpine3.17 as dependencies
+
+WORKDIR /dependencies
 
 RUN apk update
+RUN apk add python3 gcc g++ make --update-cache
 
-USER node
-RUN mkdir -p /home/node/wepublish
-WORKDIR /home/node/wepublish
+WORKDIR /dependencies
+COPY libs/api/prisma/schema.prisma ./libs/api/prisma/schema.prisma
+COPY .npmrc package.json package-lock.json ./
+RUN npm ci
 
-COPY --chown=node:node . .
 
-RUN rm -rf .env
-RUN npm ci
-RUN npm run build
+FROM node:16-alpine3.17 as builder
+WORKDIR /build
+COPY . .
+RUN rm -rf .env*
+COPY --from=dependencies /dependencies .
+RUN npm run build --prod
 
+
+FROM node:16-alpine3.17
```

Comment:
couldnt we use a slim package here?

Reply:
We also miss cleaning up the apt-get cache, should be a few hundred MBs too, no?

Reply:
> couldnt we use a slim package here?

Size was 2mb diff and actually npm ci command took longer 250 slim vs 150 alpine secs locally...